### PR TITLE
PTV HM and cleaning hm directory

### PR DESCRIPTION
**Pull request by Ashley-Pettit**
Tried to make PTV nice :P

### CODE REVIEW

File: Ashley_Pettit/18thMarchHWPTV.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+function initialize(){
+	var destination	= prompt("Where are you travelling to?");
+	var origin = prompt("Where are you right now?");
+
+  if (isinputvalid(destination,origin)) {
```
Comment (mattswann) [inline Ashley_Pettit/18thMarchHWPTV.js:5]: Nice job. This is definitely a very clever way of dealing with user input.

File: Ashley_Pettit/18thMarchHWPTV.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+			console.log("Once at Richmond you need to take the " + routeOfDestination + " towards " + destination + " for " + stopsFromRichmond + " stop(s) and you will arive at " + destination + ".");
+	  } 
+	}
+	else {
+		alert("I didn't understand your input. Lets try again");
+		initialize();
+	}
+}
+
+function isinputvalid(destination, origin) {
```
Comment (mattswann) [inline Ashley_Pettit/18thMarchHWPTV.js:25]: love it

File: Ashley_Pettit/18thMarchHWPTV.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+
+stationsAlamein = ["Flinders Street", "Richmond", "East Richmond", "Burnley", "Hawthorn", "Glenferrie"];
+stationsGlenWaverly = ["Flagstaff", "Melbourne Central", "Parliament", "Richmond", "Kooyong", "Tooronga"];
+stationsSandringham = ["Southern Cross", "Richmond", "South Yarra", "Prahran", "Windsor"];
+allStations = stationsAlamein.concat(stationsGlenWaverly).concat(stationsSandringham); //Used to check if entered stations are valid
+
+initialize();
+
+// Tests entering gibberish
```
Comment (mattswann) [inline Ashley_Pettit/18thMarchHWPTV.js:75]: I like that you have included these extra notes. Really awesome job. Well done.

**Comment by mattswann** [comment]
Awesome work on PTV. I like that you have included comments for all your functions.

**Comment by Ashley-Pettit** [comment]
Cheers bud :)

Cheers,

Ashley Pettit

**Pull request merged**